src/containers/Modal.tsx

```diff
@@ -0,0 +1,72 @@
+import { closeModal, completeOnBoarding } from '@actions'
+import { RootState } from '@store';
+import * as React from 'react'
+import { connect } from 'react-redux'
+
+import * as onboarding from '@images/onboarding_stickers_Artboard.svg'
+
+const mapStateToProps = (state: RootState) => ({
+  isModalActive: state.modals.modalDisplay,
+  hasOnBoarded: state.settings.onBoarding
+})
+
+const mapDispatchToProps = (dispatch) => ({
+  closeModal: () => (
```

Comment:
I'm pretty sure that if you don't have to do anything special with the function you can just provide the action names without having to wrap each one in the dispatch closure, eg.

```typescript
const mapDispatchToProps = {
    closeModal,
    completeOnBoarding,
}
```

Reply:
fixed